sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -3049,8 +3052,7 @@ void simd_obj_impl<T, N, T1, SFINAE>::copy_from(
 
 template <typename T, int N, class T1, class SFINAE>
 template <typename AccessorT, typename Flags, int ChunkSize, typename>
-ESIMD_INLINE EnableIfAccessor<AccessorT, accessor_mode_cap::can_read,
-                              sycl::access::target::device, void>
+ESIMD_INLINE EnableIfAccessor<AccessorT, accessor_mode_cap::can_read, void>
```

Comment:
I think this allows local accessors now, right. We have
```
template <typename T, accessor_mode_cap_val_t Capability, typename RetT>
using EnableIfAccessor =
    std::enable_if_t<detail::is_accessor_with_v<T, Capability>, RetT>;
```

and `is_accessor_with_v` is 
```
template <typename T, accessor_mode_cap_val_t Capability>
inline constexpr bool is_accessor_with_v =
    is_device_accessor_with_v<T, Capability> ||
    is_local_accessor_with_v<T, Capability>;
```

So won't local accessor be allowed because of the `||`? Or did I misunderstand? I assume we don't want to allow local accessors here.